spring-grpc-spring-boot-autoconfigure/src/test/java/org/springframework/grpc/server/GrpcServerAutoConfigurationTests.java

```diff
@@ -89,7 +94,22 @@ void serverFactoryAutoConfiguredAsExpected() {
 			.run((context) -> assertThat(context).getBean(DefaultGrpcServerFactory.class)
 				.hasFieldOrPropertyWithValue("address", "myhost")
 				.hasFieldOrPropertyWithValue("port", 6160)
-				.hasFieldOrPropertyWithValue("serverConfigurers", List.of()));
+				.hasFieldOrPropertyWithValue("serverBuilderCustomizers", List.of())
```

Comment:
**[UNRELATED_TO_THIS_PR]** I should have included this in previous proposal as it verifies the bindable services were in fact consulted to build the list of service defs.